pkg/ccl/changefeedccl/alter_changefeed_stmt.go

```diff
@@ -227,11 +227,8 @@ func alterChangefeedPlanHook(
 		newPayload.Details = jobspb.WrapPayloadDetails(newDetails)
 		newPayload.Description = jobRecord.Description
 		newPayload.DescriptorIDs = jobRecord.DescriptorIDs
-		newExpiration, err := newOptions.GetPTSExpiration()
-		if err != nil {
-			return err
-		}
-		newPayload.MaximumPTSAge = newExpiration
+		newPayload.MaximumPTSAge = jobRecord.MaximumPTSAge
```

Comment:
`jobRecord.MaximumPTSAge` will be populated correctly by the `createChangefeedJobRecord` call above